### PR TITLE
[DEL] remove useless assert

### PR DESCRIPTION
**Pull request by kaydxh**
remove useless assert, keep code clean.

### CODE REVIEW

File: daemon/graphdriver/copy/copy_test.go [hunk 1]
```diff
@@ -53,7 +53,6 @@ func TestCopyDir(t *testing.T) {
 		}
 
 		dstPath := filepath.Join(dstDir, relPath)
-		assert.NilError(t, err)
```
Comment (kaydxh) [inline daemon/graphdriver/copy/copy_test.go:56]: the assert is useless, just del it

**Comment by AkihiroSuda** [comment]
Please sign off the commit with full name

**Comment by kaydxh** [comment]
remove useless assert, keep code clean

**Review by AkihiroSuda — APPROVED** [review]

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**